resources/views/authors/show.blade.php

```diff
@@ -1,29 +1,36 @@
 @extends('layouts.app')
 
 @section('content')
-    <div class="container">
-        <section class="gistlog-author-block">
-            <a href="https://github.com/{{ $author->username }}" class="profile-pic">
-                <img src="{{ $author->avatarUrl }}">
+    <div class="container mx-auto gistlog__container my-8">
+        {{-- Avatar --}}
+        <div class="avatar">
+            <a href="https://github.com/{{ $author->username }}" target="_blank">
+                <img src="{{ $author->avatarUrl }}" alt="{{ $author->username }} - {{ config('app.name') }}">
             </a>
-            <h1 class="gistlog__title">{{ $author->name }}</h1>
-            <span class="gistlog__author">({{ "@" . $author->username }})</span>
-        </section>
+        </div>
+        <div class="gistlog py-8 sm:px-8">
+            <section class="my-8 px-4 sm:px-8 my-8">
+                <h1 class="gistlog__title">{{ $author->name }}</h1>
+                <div class="font-light mx-auto table">
+                    <span class="font-bold text-grey no-underline">{{ '@' . $author->username }}</span>
+                </div>
 
-        <section>
-            <ul class="list-unstyled">
-                @foreach ($author->gists as $gist)
-                    <li class="gistlog-article">
-                         <a href="/{{ $author->username }}/{{ $gist->id }}" class="gistlog-article__title">{{ $gist->title }}</a>
-                        <span class="gistlog-article__timestamp">
-                            Posted {{ $gist->createdAt->diffForHumans() }}
-                        </span>
-                        <p class="gistlog-article__summary">
-                            {{ $gist->getPreview() }}&hellip;
-                        </p>
-                    </li>
-                @endforeach
-            </ul>
-        </section>
+                <section class="mt-8 pt-6 gistlog__content">
```

Comment:
Same question here about `gistlog__content` and `mt-8` and `pt-6`.

Basically in these instances we should:

1. If it's really simple and only used once, just use Tailwind classes and skip the custom class
2. If it's complicated, used multiple places, use a custom class
3. if it's complicated AND displays different depending on the context, use a custom class + tailwind modifier classes per context